src/views/supplier/request-accepted.njk

```diff
@@ -0,0 +1,57 @@
+{% extends "page.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l">Request ID: RR147</span>
+    <h1 class="govuk-heading-xl">
+      Accepted request
+    </h1>
+  </div>
+</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m govuk-!-static-margin-bottom-1">Decision made</h2>
+    <p class="govuk-body">2 May 2023</p>
+
+    <h2 class="govuk-heading-m govuk-!-static-margin-bottom-1">Feedback provided</h2>
+    <p class="govuk-body">We are in a good place to start the MoU but will need to set up a meeting to discuss the finer details.</p>
+
+    <h2 class="govuk-heading-m govuk-!-static-margin-bottom-1">Notes</h2>
+    <p class="govuk-body">I spoke to George Heatherfield from legal who confirmed the correct legal gateway.</p>
+
+
+    <h2 class="govuk-heading-m govuk-!-static-margin-bottom-1">What happens next</h2>
+    <p class="govuk-body">The Department for Education have been told that this request was accepted.</p>
+    <p class="govuk-body">Contact <a href="#1">data-management@dfe.gov.uk</a> to set up the data share arrangement.</p>
```

Comment:
```suggestion
    <p class="govuk-body">Contact <a class="govuk-link" href="#1">data-management@dfe.gov.uk</a> to set up the data share arrangement.</p>
```

Thanks Gabi 🙂 I would just add the govuk-link class to the email anchor tag but other than that looks good 🎉

Reply:
Thanks @madz-lw I've added now :D